tests/test_phonebook.py

```diff
@@ -1,165 +1,272 @@
-
-membership: models_phonebook.Membership
-membership_to_delete_admin: models_phonebook.Membership
-membership_to_delete_president: models_phonebook.Membership
-membership_to_delete_simple: models_phonebook.Membership
+association1: models_phonebook.Association
+association2: models_phonebook.Association
+association3: models_phonebook.Association
 
+membership1: models_phonebook.Membership
+membership2: models_phonebook.Membership
+membership3: models_phonebook.Membership
+membership4: models_phonebook.Membership
+membership5: models_phonebook.Membership
+membership6: models_phonebook.Membership
 phonebook_user_BDE: models_core.CoreUser
 phonebook_user_president: models_core.CoreUser
 phonebook_user_simple: models_core.CoreUser
+phonebook_user_simple2: models_core.CoreUser
+phonebook_user_simple3: models_core.CoreUser
+
+association1_group: models_core.CoreGroup
+association2_group: models_core.CoreGroup
+
+association2_group2: models_phonebook.AssociationAssociatedGroups
 
-token_BDE: str
-token_president: str
-token_simple: str
+token_admin: str = ""
+token_BDE: str = ""
+token_president: str = ""
+token_simple: str = ""
 
 
 @pytest_asyncio.fixture(scope="module", autouse=True)
-async def init_objects() -> None:
+async def init_objects(client: TestClient):
     global phonebook_user_BDE
+    global token_BDE
+    global phonebook_user_president
+    global token_president
+    global phonebook_user_simple
+    global phonebook_user_simple2
+    global phonebook_user_simple3
+    global token_simple
+    global token_admin
+
+    global association1
+    global association2
+    global association3
+
+    global membership1
+    global membership2
+    global membership3
+    global membership4
+    global membership5
+    global membership6
+
+    global association1_group
+    global association2_group
+
+    global association2_group2
+
     phonebook_user_BDE = await create_user_with_groups(
         [GroupType.student, GroupType.BDE],
     )
-
-    global token_BDE
     token_BDE = create_api_access_token(phonebook_user_BDE)
 
-    global phonebook_user_president
     phonebook_user_president = await create_user_with_groups([GroupType.student])
-
-    global token_president
     token_president = create_api_access_token(phonebook_user_president)
 
-    global phonebook_user_simple
     phonebook_user_simple = await create_user_with_groups([GroupType.student])
-
-    global token_simple
     token_simple = create_api_access_token(phonebook_user_simple)
 
-    global association
-    association = models_phonebook.Association(
+    phonebook_user_simple2 = await create_user_with_groups([GroupType.student])
+    phonebook_user_simple3 = await create_user_with_groups([GroupType.student])
+
+    phonebook_user_admin = await create_user_with_groups([GroupType.admin])
+    token_admin = create_api_access_token(phonebook_user_admin)
+
+    association1_group = models_core.CoreGroup(
+        id="1",
```

Comment:
Using str(UUID) would make a future migration from string to uuid type easier

Reply:
UUID mess up everything, good luck